backend/src/core/files.py

```diff
@@ -0,0 +1,42 @@
+import os
+import secrets
+
+import aiofiles
+from fastapi import UploadFile
+from fastapi.responses import FileResponse
+
+from src.core.config import settings
+
+
+def generate_name(file: UploadFile):
+    file_type = file.filename.split(".")[-1]
+    filename = f"{secrets.token_hex(nbytes=12)}.{file_type}"
+    while os.path.exists(os.path.join(settings.FILES_PATH, filename)):
+        filename = f"{secrets.token_hex(nbytes=12)}.{file_type}"
```

Comment:
Avoid copy-pasting `f"{secrets.token_hex(nbytes=12)}.{file_type}"`. Extract it to a function like `generate_name`

Reply:
Done